gui-apps/rofi-wayland/rofi-wayland-1.7.5.ebuild

```
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S="${WORKDIR}"/rofi-"${PV}"+wayland1
```

${PV} will never contain spaces, so:
```suggestion
S="${WORKDIR}"/rofi-${PV}+wayland1
```

---

or
```sh
S="${WORKDIR}/rofi-${PV}+wayland1"
```
works as well (and IMO is better practice).

---

Why though? I mean what Sam suggested. 

I usually do what NRK says or I put every variable in doubles quotes. Is that wrong?